Update docs syntax

jeffjennings opened the pull request:
This PR updates the function calls in the docs to be consistent with current code and makes a minor related change to figure generation (adds `figsize` variable to all figures).

rbooth200 commented on docs/quickstart.rst:
```diff
@@ -136,21 +136,20 @@ but we're going to mostly avoid those here,
 
 First import some basic stuff from frank and load the data
-(again using the DSHARP observations of AS 209, available as a UVTable
+(again using the DSHARP observations of AS 209, available as a UVTable - with some reasonable time- and channel-averaging -
```
Please add the details...

jeffjennings replied:
I think the time averaging for the AS 209 UVTable is 10 s (or 30 s?) and the channel averaging 1 channel per spw (spectral window). @mtazzari is that correct? 

(I just wanted to add this detail in case people are comparing directly to the MS Table on the DSHARP website, but don't have to.)

jeffjennings replied:
...I found the averaging used (stated in the frank ms). Updated in b49c8b1

jeffjennings commented:
> Thanks Jeff. What changed n "fitting_procedure.ipynb"

I know it's annoying isn't it that the diff shows all the raw .ipynb code. I just changed the function calls to have the correct arguments for the current version of the code, and I removed a few imports that are no longer used as a result.

rbooth200 commented:
> I know it's annoying isn't it that the diff shows all the raw .ipynb code. I just changed the function calls to have the correct arguments for the current version of the code, and I removed a few imports that are no longer used as a result.

Fine by me.

jeffjennings commented:
I think this is ready to merge now unless you have other change requests.
